pages/404.html

```diff
@@ -5,9 +5,40 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="../css/main.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src='../assets/js/main.js'></script>
     <title>Document</title>
 </head>
 <body>
```

Comment:
Give this a class so all the styles within this can be nested within that class.

Reply:
Done